src/neo_usboard_v2_node.cpp

```diff
@@ -43,7 +46,49 @@ class ROS_Node : public neo_usboard_v2::ROS_NodeBase {
 	void handle(std::shared_ptr<const pilot::usboard::USBoardData> value) override
 	{
 		// TODO: convert and publish on ROS
-	}
+		neo_msgs::USBoardV2 usBoard;
+		int i = 0;
+		
+		for(const auto& sensor : config->sensor_config)
+		{
+			std_msgs::Header USRangeHeader;
+			sensor_msgs::Range USRangeMsg;
+			if(sensor.active)
+			{
+				usBoard.active[i] = 1;
+				// Creating rostopics only if they are active 
+				topicPub_USRangeSensor[i] = nh.advertise<sensor_msgs::Range>("/USBoardV2/Sensor"+std::to_string(i),1);
```

Comment:
Here lower case again: `/usboard_v2/sensorX`

Reply:
Also you should not call advertise() every time, only the first time.

Reply:
Ideally in the USBoardConfig callback, see `void handle(std::shared_ptr<const pilot::usboard::USBoardConfig> value)`